flink-test-utils/src/main/java/org/apache/flink/test/util/TestBaseUtils.java

```diff
@@ -451,6 +454,53 @@ protected static File asFile(String path) {
 			assertEquals(extectedStrings[i], resultStrings[i]);
 		}
 	}
+	
+	// --------------------------------------------------------------------------------------------
+	// Comparison methods for tests using sample
+	// --------------------------------------------------------------------------------------------
+	
+	public static <T> void containsResultAsTuples(List<T> result, String expected) {
+		isExpectedContainsResult(result, expected, true);
+	}
+	
+	public static <T> void containsResultAsText(List<T> result, String expected) {
+		isExpectedContainsResult(result, expected, false);
+	}
+	
+	private static <T> void isExpectedContainsResult(List<T> result, String expected, boolean asTuple) {
```

Comment:
Can we get comments explaining the functionality of this and `containsResultAsText`?